map2loop/project.py

```diff
@@ -190,35 +199,38 @@ def __init__(
             self.map_data.set_filename(Datatype.DTM, dtm_filename)
         if fault_orientation_filename != "":
             self.map_data.set_filename(Datatype.FAULT_ORIENTATION, fault_orientation_filename)
+        
+        
         if config_filename != "":
             if clut_file_legacy:
                 print(
                     "DEPRECATION: Legacy files are deprecated and their use will be removed in v3.2"
                 )
             self.map_data.set_config_filename(config_filename, legacy_format=clut_file_legacy)
+        
         if config_dictionary != {}:
             self.map_data.config.update_from_dictionary(config_dictionary)
+        
         if clut_filename != "":
             self.map_data.set_colour_filename(clut_filename)
 
         # Load all data (both shape and raster)
         self.map_data.load_all_map_data()
 
         # If flag to save out data is check do so
+        tmp_path = pathlib.Path(tempfile.mkdtemp())
+
         if save_pre_checked_map_data:
             self.map_data.save_all_map_data(tmp_path)
 
         # Populate the stratigraphic column and deformation history from map data
         self.stratigraphic_column.populate(self.map_data.get_map_data(Datatype.GEOLOGY))
         self.deformation_history.populate(self.map_data.get_map_data(Datatype.FAULT))
-
-        # Set default minimum fault length to 5% of the longest bounding box dimension
-        bounding_box = self.map_data.get_bounding_box()
-        largest_dimension = max(
-            bounding_box["maxx"] - bounding_box["minx"], bounding_box["maxy"] - bounding_box["miny"]
-        )
-        self.deformation_history.set_minimum_fault_length(largest_dimension * 0.05)
-
+        
+        # only define the minimum fault length if there are faults
+        if len(self.map_data.get_map_data(Datatype.FAULT))!=0:
```

Comment:
Isn't the fault minimul length stored in the config?

Reply:
yes, this was redundant. removed in https://github.com/Loop3D/map2loop/pull/112/commits/23873fa16c5d9a7d41fdfff22d1ecc9461c31fb4